### ISSUE TITLE
Automatic notification for answers

### ISSUE THREAD

**Issue by Schoaf** [Schoaf/qa-mattermost-notifications#3]
When a question has been answered the first time, a notification should be posted into Mattermost so that all people know it has already been answered. This could potentially been done by referencing the post of the question - like a comment.

**Comment by Schoaf** [comment]
Now with the new Mattermost, we could mark the original question as answered by replying with an emoji ( "+:ballot_box_with_check:" ).

**Comment by Schoaf** [comment]
Solving this issue would require 
* AskAgfa to store the post ID together with the channels it posted to (because the settings could change in between) and the question it relates to.
* A user account for the bot that has access to all channels it posts to (complicated to set up)
* Then it would have to login, get the specific channels and the posts and update those posts.

There is another idea to solve this problem by letting the user update the post:
Relates to #10 where a simpler solution has been implemented.

**Comment by rklec** [comment]
Is there any news for this?

**Comment by Schoaf** [comment]
No. Unfortunately the plug-in is not under active development any more. I dont even have a test setup right now.
But if you are willing to contribute code I would gladly support you...
